src/Microsoft.ApplicationInsights.AspNetCore/project.json

```diff
@@ -21,83 +21,83 @@
     "delaySign": true,
     "xmlDoc": true
   },
-  "dependencies": {
-    "Desktop.Analyzers": {
-        "version": "1.2.0-beta2",
-        "type": "build"
+    "dependencies": {
```

Comment:
nitpick: Please unindent the extra two spaces so it doesn't appear changed in the diff when it didn't really change.  Or alternately do a CTRL-K CTRL-D and reformat the whole file so VS doesn't keep messing up the formatting.  I have no idea why some templates use two spaces and some use four but it makes for awkward diffs later when VS isn't consistent with its templates.  Not your fault of course.